ninio-snmp/src/main/java/com/davfx/ninio/snmp/encryption/SecurityProtocols.java

```diff
@@ -0,0 +1,39 @@
+package com.davfx.ninio.snmp.encryption;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public final class SecurityProtocols {
+
+    private static final List<AuthProtocol> AUTH_GENERIC_LIST = ImmutableList.of(new AuthMD5(), new AuthSHA1(), new AuthSHA256());
+    private static final List<PrivacyProtocol> PRIVACY_GENERIC_LIST = ImmutableList.of(new PrivDES(), new PrivAES(), new PrivAES256());
```

Comment:
This comment is beyond the scope of this file but it seems to be the best place for it: 
I think enums would be better for `AuthProtocol` and `PrivacyProtocol`, for two reasons:
- 1 centralized place to read/add the list of available protocols (impossible to forget to add, everybody uses the `<enum>.values()` to iterate)
- autocompletion to quickly see/use the possible values